src/routes/index.js

```diff
@@ -61,6 +64,10 @@ export default function Router() {
             <Route path=":projectId">
               <Route path="admin" element={<RC element={<ProjectAdmin />} />} />
               <Route index element={<RC element={<Project />} />} />
+              <Route
```

Comment:
Change this so that we use relative paths, not absolute paths:
```js
<Route path="admin">
  <Route path="members" element={<RC element={<Members />} />} />
  <Route index element={<RC element={<ProjectAdmin />} />} />
</Route>
```